components/markdown_image_expand/markdown_image_expand.tsx

```diff
@@ -9,7 +9,7 @@ export type Props = {
     children: React.ReactNode;
     isExpanded: boolean;
     postId: string;
-    onToggle: (isExpanded: boolean) => void;
+    onToggle?: (isExpanded: boolean) => void;
```

Comment:
Is this ok to be optional now that we are passing 'onImageHeightChanged' correctly?

Reply:
We are passing it on posts and attachments. Markdown might be used in more places. If we do not make it optional, we might find unexpected crashes in other places.